man/coap_handler.txt.in

```diff
@@ -207,6 +207,32 @@ COAP_EVENT_TCP_FAILED         0x1003
 COAP_EVENT_SESSION_CONNECTED  0x2001
 COAP_EVENT_SESSION_CLOSED     0x2002
 COAP_EVENT_SESSION_FAILED     0x2003
+/**
+ * (Q-)BLOCK receive errors
+ */
+COAP_EVENT_PARTIAL_BLOCK     = 0x3001,
+
+/**
```

Comment:
I think the comments should be used in the source code only. Asciidoc allows for more readable text markup outside the code environment.

Reply:
I removed the detailed comments for the specific event types, but kept the comments that are used to group the events  into categories for consistency reasons (as they were also included for the other categories, see lines 191, 198, 204).